Oh, brilliant, there seems to be another issue, that brakes the i686 builds, https://travis-ci.org/MacPython/astropy-wheels/jobs/274403410#L921

```
=================================== FAILURES ===================================

__________________________ test_dtype_bytes_or_chars ___________________________

    def test_dtype_bytes_or_chars():

        assert misc.dtype_bytes_or_chars(np.dtype(np.float64)) == 8

>       assert misc.dtype_bytes_or_chars(np.dtype(object)) == (8 if NUMPY_LT_1_10 else None)

E       AssertionError: assert 4 == 8

E        +  where 4 = <function dtype_bytes_or_chars at 0xf3ce6d14>(dtype('O'))

E        +    where <function dtype_bytes_or_chars at 0xf3ce6d14> = misc.dtype_bytes_or_chars

E        +    and   dtype('O') = <type 'numpy.dtype'>(object)

E        +      where <type 'numpy.dtype'> = np.dtype

/venv/local/lib/python2.7/site-packages/astropy/utils/tests/test_misc.py:122: AssertionError
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -32,44 +32,44 @@ matrix:
       env:
         - MB_PYTHON_VERSION=2.7
         - UNICODE_WIDTH=16
-    - os: linux
-      env:
-        - MB_PYTHON_VERSION=2.7
-        - PLAT=i686
-    - os: linux
-      env:
-        - MB_PYTHON_VERSION=2.7
-        - PLAT=i686
-        - UNICODE_WIDTH=16
-    - os: linux
-      env:
-        - MB_PYTHON_VERSION=3.4
+#    - os: linux
+#      env:
+#        - MB_PYTHON_VERSION=2.7
+#        - PLAT=i686
+#    - os: linux
+#      env:
+#        - MB_PYTHON_VERSION=2.7
+#        - PLAT=i686
+#        - UNICODE_WIDTH=16
     - os: linux
       env:
         - MB_PYTHON_VERSION=3.4
-        - PLAT=i686
+#    - os: linux
+#      env:
+#        - MB_PYTHON_VERSION=3.4
+#        - PLAT=i686
     - os: linux
       env:
         - MB_PYTHON_VERSION=3.5
         - NP_BUILD_DEP=numpy==1.9.3
         - NP_TEST_DEP=numpy==1.9.3
-    - os: linux
-      env:
-        - MB_PYTHON_VERSION=3.5
-        - PLAT=i686
-        - NP_BUILD_DEP=numpy==1.9.3
-        - NP_TEST_DEP=numpy==1.9.3
+#    - os: linux
+#      env:
+#        - MB_PYTHON_VERSION=3.5
+#        - PLAT=i686
+#        - NP_BUILD_DEP=numpy==1.9.3
+#        - NP_TEST_DEP=numpy==1.9.3
     - os: linux
       env:
         - MB_PYTHON_VERSION=3.6
         - NP_BUILD_DEP=numpy==1.11.3
         - NP_TEST_DEP=numpy==1.11.3
-    - os: linux
-      env:
-        - MB_PYTHON_VERSION=3.6
-        - PLAT=i686
-        - NP_BUILD_DEP=numpy==1.11.3
-        - NP_TEST_DEP=numpy==1.11.3
+#    - os: linux
+#      env:
+#        - MB_PYTHON_VERSION=3.6
+#        - PLAT=i686
+#        - NP_BUILD_DEP=numpy==1.11.3
+#        - NP_TEST_DEP=numpy==1.11.3
     - os: osx
       language: generic
       env: MB_PYTHON_VERSION=2.7

